src/QualificationRunner.Core/Services/QualificationRunner.cs

```diff
@@ -420,7 +420,7 @@ private IReadOnlyList<Input> retrieveInputs(dynamic qualificationPlan)
 
          var section = sections.FirstOrDefault(x => x.Id == sectionId);
          if (section != null)
-            return currentLevel;
+            return currentLevel + 1;

```

Comment:
Can you add a comment to explain why you do +1?

Reply:
done